.github/pull_request_template.md

```diff
@@ -1,18 +1,22 @@
-*Before you submit this PR make sure you have added an entry to **.changelog** file.*
+# Some hints
+
+Consider adding an entry to the `.changelog` file in the toplevel directory.
+It is a hidden file with a dot at the beginning.
+
+Add Description, Target branches, and related Links below the following section titles.
+
+In the Description, just enter a summary of why you created this PR (and, if available, add any relevant diagram).
 
 # Description
 
 Short summary of why you created this PR (if you added documentation, please add any relevant diagram).
 
 # Target branches
 
-* Which documentation version does this PR apply to?
-* Does this content need to be backported? If this is the case, create an issue for tracking it and add the link to this PR.
+* Which product version this PR applies to (Uyuni YYYY.MM, SUMA 4.3, SUMA MU X.Y.Z, or development version) 
```

Comment:
Do you really distinguish between the various Uyuni versions and master (which surely is the dev version)?

Reply:
Sometimes yes ;)  If you create a PR, you probably know the version and branch.  This info asked for here is a safety net, thus we can enter ifdefs where needed.  Uyuni version probably is superfluous. I'll remove it.